funnel/views/project.py

```diff
@@ -89,17 +91,13 @@ class ProfileProjectView(ProfileViewMixin, UrlForView, ModelView):
     @requires_roles({'admin'})
     def new_project(self):
         form = ProjectForm(model=Project, parent=self.obj)
-        # Profile URLs:
-        # HasGeek: https://hasgeek.com/rootconf (no /)
-        # Talkfunnel: https://rootconf.talkfunnel.com/ (has /)
-        form.name.prefix = self.obj.url_for(_external=True)
-        if not form.name.prefix.endswith('/'):
-            form.name.prefix += '/'
+
         if request.method == 'GET':
             form.timezone.data = current_app.config.get('TIMEZONE')
         if form.validate_on_submit():
             project = Project(user=current_auth.user, profile=self.obj)
             form.populate_obj(project)
+            project.name = make_name(project.title)
```

Comment:
Doesn't `Project` use `BaseScopedNameMixin`? This can be simply `project.make_name()`.